data/scenarios/Testing/373-drill.yaml

```diff
@@ -50,7 +61,7 @@ world:
     ┌─────┐                                                                ~~
     │IAAT~                                                                ~L~
     │..AAA│                                                                ~~
-    │....C│                                                                 ~
+    │Ω...C│                                                                 ~
     └─────┘                                                                 ~
 entities:
   - name: copper vein
```

Comment:
I think we need a regular mountain entity too, otherwise, this can randomly fail.

Some mountain recipes do not give you rocks, e.g. `mountain tunnel`+`lodestone` which is very common now.

Reply:
I changed one of the mountains to a boulder which is guaranteed to yield `rock` when drilled.